azurerm/internal/services/eventhub/eventhub_namespace_resource.go

```diff
@@ -79,6 +79,12 @@ func resourceArmEventHubNamespace() *schema.Resource {
 				Default:  false,
 			},
 
+			"zone_redundant": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
```

Comment:
can we add an acceptance test covering setting/updating this field? [here's an example of doing that](https://github.com/terraform-providers/terraform-provider-azurerm/blob/28bfd04fd19e0637c889989298ec984c4e83b4c5/azurerm/internal/services/eventhub/tests/eventhub_namespace_resource_test.go#L156-L182)

Reply:
Added a test, please have a look?